<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionMenu</class>
 <widget class="QDialog" name="ActionMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>187</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Iventory Option Panel</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ChangeInventory">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Change Inventory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_CustomInput">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Custom Input</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ShoppingList">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Shopping List</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>197</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
